src/components/Todos/DoinItGrid.js

```diff
@@ -0,0 +1,22 @@
+import { TodoList } from '../UI/TodoCard/TodoList.styled';
+import { TodoLi } from '../UI/TodoCard/TodoListItem.styled';
+import { Container } from '../UI/Grid/Container';
+import Todos from './Todos';
+import { ListHeadline } from '../UI/TodoCard/ListHeadline.styled';
+
+export default function DoinItGrid({ todos }) {
+	return (
+		<Container>
+			<ListHeadline>DoingIT</ListHeadline>
+			<TodoList role="list">
+				{todos
+					.filter(todos => todos.status === 'doingIt')
```

Comment:
Since .filter() goes through every single todo object, it is good to name it with a singular.
```suggestion
					.filter(todo => todo.status === 'doingIt')
```